app/actors/sufia/create_with_files_actor.rb

```diff
@@ -2,6 +2,7 @@ module Sufia
   # Creates a work and attaches files to the work
   class CreateWithFilesActor < CurationConcerns::Actors::AbstractActor
     def create(attributes)
+      set_publication_state
       self.uploaded_file_ids = attributes.delete(:uploaded_files)
       validate_files && next_actor.create(attributes) && attach_files
```

Comment:
@hortongn It makes sense to put your change in this actor, yes. We may eventually want to break this functionality out into its own actor but it's early days to work on that abstraction, I feel. You may want to add this to the chain of operations on line 7:

```ruby
validate_files && mark_inactive && next_actor.create(attributes) && attach_files
```